test/framework/test_manager.ts

```diff
@@ -0,0 +1,112 @@
+/// FIXME(jalextowle): I have a plan for making this file type safe (with useful
+/// default types). This will make writing tests significantly easier.
+import 'mocha';
+
+import { LoggingConfig, setupApiAsync, teardownApiAsync } from '../utils/deployment';
+
+import * as actions from './actions';
+import * as assertions from './assertions';
+
+/**
+ * Constructs the default `TestManager` class.
+ * @returns TestManager Returns a `TestManager` that has been given default actions
+ *          and assertions.
+ */
+export function defaultTestManager(): TestManager {
+    return new TestManager(
+        {
+            apiGetRequestAsync: actions.apiGetRequestAsync,
+        },
+        {
+            assertFieldEqualsAsync: assertions.assertFieldEqualsAsync,
```

Comment:
nit: would probably rename this to `assertEquals` for conciseness, anticipating a whole slew of `assertGreaterThan`, `assertLessThan`, etc.